<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDockWidget" name="dockWidget_edge">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>300</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>481</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>481</height>
     </size>
    </property>
    <property name="windowTitle">
     <string>滤镜</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_5">
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>灰      度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_gray">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>亮      度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_light">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>锐      化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_sharpness">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>对  比  度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Contrast">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>伽  马  值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_gamma">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>饱  和  度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Saturation">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>二  值  化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_binaryzation">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>边缘  检测</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_detection">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>均值  滤波</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_filter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>782</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>760</width>
        <height>510</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>510</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>760</width>
        <height>510</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>760</width>
        <height>510</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>510</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>760</width>
        <height>510</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_18"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>262</width>
      <height>559</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QDockWidget" name="dockWidget_default">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>250</height>
        </size>
       </property>
       <property name="windowTitle">
        <string>默认设置</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents_7">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="btn_openpic">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>打开图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_savepic">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>保存图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QPushButton" name="btn_videochange">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>视频界面</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_imgchange">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>图片处理</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QPushButton" name="btn_oldpic">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>原图显示</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_default">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>恢复默认</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDockWidget" name="dockWidget_color">
       <property name="minimumSize">
        <size>
         <width>260</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>300</height>
        </size>
       </property>
       <property name="windowTitle">
        <string>色彩调整</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents_6">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>R值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_red">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>G值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_green">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>B值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_blue">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="btn_turnleft">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>向左旋转</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_turnright">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>向右旋转</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="btn_mirror">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>镜像翻转</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_turntop">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>上翻转</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1344</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>效果</string>
    </property>
    <addaction name="actionbright"/>
    <addaction name="actionbianyuan"/>
    <addaction name="actionjunzhi"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>三原色</string>
    </property>
    <addaction name="actionRed"/>
    <addaction name="actionGreen"/>
    <addaction name="actionBlue"/>
   </widget>
   <widget class="QMenu" name="menu_fanzhuan">
    <property name="title">
     <string>翻转提示</string>
    </property>
    <addaction name="actionturntop"/>
    <addaction name="actionturnmirror"/>
    <addaction name="actionturnright"/>
    <addaction name="actionturnleft"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_fanzhuan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsave">
   <property name="text">
    <string>保存(CTRL+S)</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>打开(CTRL+O)</string>
   </property>
  </action>
  <action name="actionhuidu">
   <property name="text">
    <string>灰度()</string>
   </property>
  </action>
  <action name="actionbright">
   <property name="text">
    <string>亮度</string>
   </property>
  </action>
  <action name="actionbianyuan">
   <property name="text">
    <string>边缘检测</string>
   </property>
  </action>
  <action name="actionjunzhi">
   <property name="text">
    <string>均值滤波</string>
   </property>
  </action>
  <action name="actionbaohedu">
   <property name="text">
    <string>饱和度</string>
   </property>
  </action>
  <action name="actionduibidui">
   <property name="text">
    <string>对比度()</string>
   </property>
  </action>
  <action name="actionruihua">
   <property name="text">
    <string>锐化()</string>
   </property>
  </action>
  <action name="actiongamma">
   <property name="text">
    <string>伽马值()</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>R</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>G</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>B</string>
   </property>
  </action>
  <action name="actionturnright">
   <property name="text">
    <string>向右转(CTRL+→)</string>
   </property>
  </action>
  <action name="actionturnleft">
   <property name="text">
    <string>向左转(CTRL+←)</string>
   </property>
  </action>
  <action name="actionturntop">
   <property name="text">
    <string>垂直翻转(CTRL+↑)</string>
   </property>
  </action>
  <action name="actionturnmirror">
   <property name="text">
    <string>镜像翻转(CTRL+↓)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
